spec/policies/placements/partnership_policy_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+describe Placements::PartnershipPolicy do
+  subject(:placements_partnership_policy) { described_class }
+
+  let(:placements_school) { create(:placements_school) }
+  let(:placements_provider) { create(:placements_provider) }
+  let!(:placements_partnership) do
+    create(
+      :placements_partnership,
+      school: placements_school,
+      provider: placements_provider,
+    )
+  end
+
+  permissions :destroy? do
```

Comment:
Do you also need to cover the `:remove?` permission? Or do you think `:destroy?` is sufficient here?

Reply:
resolved in [3602283](https://github.com/DFE-Digital/itt-mentor-services/pull/449/commits/3602283996a7608085d29cdac991157fd989e79d)